apps/api/src/notifications/providers/notifications.interface.providers.ts

```diff
@@ -4,12 +4,30 @@ type NotificationsInterfaceParams = {
   DeleteListParams: any
   AddToListParams: any
   RemoveFromListParams: any
+  GetContactsInfoParams: any
+  RemoveFromUnsubscribedParams: any
+  AddToUnsubscribedParams: any
```

Comment:
could you check if possible to use "unknown" instead of "any"?

Reply:
Done